webpack/assets/javascripts/react_app/components/Editor/components/EditorSafemodeCheckbox.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { translate as __ } from '../../../common/I18n';
+import './editorsafemodecheckbox.scss';
+
+const EditorSafemodeCheckbox = ({ isChecked, show, setSafemode }) => (
+  <React.Fragment>
+    {show && (
+      <React.Fragment>
+        <label htmlFor="safemode-rendering-checkbox">{__('Safemode')}</label>
```

Comment:
Can you add `safemode` to the `.eslintrc` skipwords list?
currently it throws a warning: `You have a misspelled word: safemode on String`